Samples/CSharp/LicensureClient.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Net.Http;
+
+namespace CSharp
+{
+    public class LicensureClient : HttpClient
+    {
+        //Access token
+        string token;
+        string baseURL;
+
+        public LicensureClient(string accessToken, string baseUrl)
```

Comment:
The clients aren't going to behave correctly in a real-world scenario. HttpClient is designed to be created once and then reused for the life of an app. The access token will have to change as the app evolves. It is also a shared object which means you cannot change the headers, instead it has to be adjusted at the call itself.

The async calls should be truly async and not reference the Console at all. I'd like to rework these samples but I don't want to hold up the docs so let's pull the samples from the PR and we'll update them separately.